public/typography.css

```diff
@@ -123,6 +123,7 @@ p,
   font-weight: 600;
 }
 
+strong,
```

Comment:
I don’t think it’s a good idea to add styles to elements. You often end up overwriting the rules when you want to use `<strong>` with different style. Instead we should just add the `typo-bold` class to the element.